packages/devtools_app/lib/src/timeline/timeline_controller.dart

```diff
@@ -415,6 +419,23 @@ class TimelineController
     return _exportController.downloadFile(encodedData);
   }
 
+  void updateSearchMatches() {
+    updateMatches(matchesForSearch(search));
+  }
+
+  List<TimelineEvent> matchesForSearch(String search) {
+    if (search == null || search.isEmpty) return [];
+    final matches = <TimelineEvent>[];
+    for (final event in data.timelineEvents) {
+      breadthFirstTraversal<TimelineEvent>(event, action: (TimelineEvent e) {
+        if (e.name.toLowerCase().contains(search.toLowerCase())) {
```

Comment:
as a very minor nit, you could lowercase the search term outside of this loop

Reply:
done